src/modules/openPopup.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-use-before-define */
+
+import { likeCountes } from './getCounts.js';
+
+export function openPopup(id) {
+  function fetchPokemonsPopup(id) {
+    fetch(`https://pokeapi.co/api/v2/pokemon/${id}`)
+      .then((response) => response.json())
+      .then((pokemonInfo) => {
+        renderPkmnPopup(pokemonInfo);
+      });
+  }
+
+  function createTypes(types, ul) {
+    types.forEach((type) => {
+      const typeLi = document.createElement('li');
+      typeLi.innerText = type.type.name;
+      ul.append(typeLi);
+    });
+  }
+
+  function createPopupImage(pokeID, containerDiv) {
+    const pokeImgContainer = document.createElement('div');
+    pokeImgContainer.classList.add('popup-image');
+
+    const pokeImage = document.createElement('img');
+    pokeImage.srcset = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/${pokeID}.png`;
+
+    pokeImgContainer.append(pokeImage);
+    containerDiv.append(pokeImgContainer);
+  }
```

Comment:
There is a function created in renderPokeImage, please reuse this function, it will improve code reusbility.